File: ScriptLibrary/MatrixEngine/Platform/NetHelper.cs
using MatrixEngine.Native;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using MatrixEngine;

namespace MatrixEngine.Platform
{
    public class NetHelper
    {
        public static void OpenUrl(string url)
        {

            switch (MatrixEngine.Engine.System.TARGET_PLATFORM)
            {
                case MatrixEngine.Engine.System.PLATFORM.WIN32:
                case MatrixEngine.Engine.System.PLATFORM.UNKNOWN:
                    global::System.Diagnostics.Process.Start(url);
                    break;
                default:
                    NativeNetHelper.OpenUrl(url);
                    break;
            }

        }
    }
}

File: MatrixEngine/Classes/Platform/Win32/NetHelper.cpp

#include "Platform/NetHelper.h"

#if(CC_TARGET_PLATFORM == CC_PLATFORM_WIN32)
 
static NetHelper *s_SharedNetHelper = NULL;

NetHelper::NetHelper()
{

}

NetHelper::~NetHelper()
{

}

void NetHelper::openUrl(const char* url)
{

}

NetHelper* NetHelper::shareNetHelper()
{
	if(!s_SharedNetHelper)
	{
		s_SharedNetHelper = new NetHelper();
	}
	return s_SharedNetHelper;
}

#endif